_layouts/qa.html

```diff
@@ -0,0 +1,26 @@
+---
+layout: default
+color: no-logo program-page
+color: dark-nav
+---
+<div class='keyline-top pad8y qa-page'>
+  <div class='limiter'>
+    <div class='pad4y col12 clearfix prose min-height'>
+      <h1>{{page.title}}</h1>
+      {% if page.subtitle %}<h2 class="abstract-subtitle">{{page.subtitle}}</h2>{% endif %}
```

Comment:
I guess having the subtitle here is not really necessary, and could be slightly confusing maybe. I would rather omit it.

Reply:
Ok, removed. There were not many subtitles anyway.